projects/aws-observability/aws-otel-collector/build/update_digest.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
```

Comment:
this looks like a copy of build/lib/image_shasum.sh

Reply:
Ah I was using `Cilium` project as a reference and didn't realize the `image_shasum.sh` did the exact same thing. I made an update to reference `build/lib/image_shasum.sh` instead.